src/gc/gcimpl.h

```diff
@@ -274,6 +274,13 @@ class GCHeap : public IGCHeapInternal
 
     virtual void DescrGenerationsToProfiler (gen_walk_fn fn, void *context);
 
+    virtual void SetDirty(void* address, size_t size);
+
+    virtual void* GetTable();
+
+    virtual bool GetGcRuntimeStructuresValid();
```

Comment:
The DAC needs access to this as well (look under src\debug\daccess), but it won't be able to call virtual methods on the GCInterface to get it. This may rather return pointer the flag that keeps track of whether the structures are valid and cache this pointer in a static on VM side, so that it works for DAC.